IoTuring/Entity/Deployments/Disk/Disk.py

```diff
@@ -103,6 +121,66 @@ def Update(self) -> None:
             attributeKey=EXTRA_KEY_DISK_FREE,
             attributeValue=usage.free,
             valueFormatterOptions=VALUEFORMATOPTIONS_DISK_GB)
+        
+        if self.configuredIo:
+            # because of this reverse parsing to get the devicename from the partition name, all the following is only linux having the partition named nvme* or sd* 
+            
+            if OsD.IsLinux():
+                devname = self.disk_partition.device.split("/")[-1]
+                disk_io = psutil.disk_io_counters(perdisk=True)[devname]
+
```

Comment:
And here is windows being different again :D
Windows Drive get reported as PhysicalDrive0 in the disk_io dict. being their real "devname"
There is an Issue about converting Driveletter to PhysicalDrive but the psutil owner gave up on it.
https://github.com/giampaolo/psutil/issues/942